PT2025/iocBoot/iocPT2025-IOC-01/config.xml

```diff
@@ -5,6 +5,10 @@
 <ioc_details>NMR Teslameter PT2025</ioc_details>
 <macros>
 <macro name="PORT" pattern="^COM[0-9]+$" description="Serial COM port" />
+<macro name="BAUD" pattern="^[0-9]+$" description="Serial communication baud rate, defaults to 9600." defaultValue="9600" hasDefault="YES" />
+<macro name="BITS" pattern="^[0-9]$" description="Serial communication number of bits, defaults to 8." defaultValue="8" hasDefault="YES" />
+<macro name="PARITY" pattern="^(even)|(odd)|(none)$" description="Serial communication parity, defaults to none." defaultValue="none" hasDefault="YES" />
```

Comment:
Shouldn't this be `^(even|odd|none)$` ? I tried `^(even)|(odd)|(none)$` in https://regex101.com/ and it seemed to match more than intended